.github/scripts/install_dependencies.sh

```diff
@@ -3,61 +3,60 @@
 sudo apt update
 
 sudo apt install -y \
-  autoconf \
-  automake \
-  bash \
-  bison \
-  binutils \
-  binutils-gold \
-  build-essential \
-  capnproto \
-  cmake \
-  ctags \
-  curl \
-  doxygen \
-  flex \
-  fontconfig \
-  gdb \
-  git \
-  gperf \
-  libcairo2-dev \
-  libcapnp-dev \
-  libgtk-3-dev \
-  libevent-dev \
-  libfontconfig1-dev \
-  liblist-moreutils-perl \
-  libncurses5-dev \
-  libx11-dev \
-  libxft-dev \
-  libxml++2.6-dev \
-  libreadline-dev \
-  tcl-dev \
-  libffi-dev \
-  perl \
-  texinfo \
-  time \
-  valgrind \
-  zip \
-  qt5-default \
-  uuid-dev \
-  default-jdk \
-  clang-format-7 \
-  g++-7 \
-  gcc-7 \
-  g++-8 \
-  gcc-8 \
-  g++-9 \
-  gcc-9 \
-  g++-10 \
-  gcc-10 \
-  g++-11 \
-  gcc-11 \
-  clang-6.0 \
-  clang-7 \
-  clang-10 \
-  libtbb-dev
 
 pip install -r requirements.txt
+    autoconf \
```

Comment:
The packages from LINE8-59 are for the ``apt install``, while the ``requirements.txt`` are only for pip install.
So there is no need to move the lines.